From 465514b0b62eb5dbd585054ea52a23993fdac564 Mon Sep 17 00:00:00 2001
From: Haixia Shi <hshi@chromium.org>
Date: Wed, 21 Jan 2015 16:14:41 -0800
Subject: [PATCH 19/24] CHROMIUM: egl/dri2: Report EXT_image_dma_buf_import with surfaceless

Surfaceless platform supports the extension, so the preprocessor check
has to be modified to include it.

BUG=none
TEST=none

Signed-off-by: Haixia Shi <hshi@chromium.org>
Signed-off-by: Tomasz Figa <tfiga@chromium.org>
---
 src/egl/drivers/dri2/egl_dri2.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/egl/drivers/dri2/egl_dri2.c b/src/egl/drivers/dri2/egl_dri2.c
index 461735fe9e37..49fcc07a6098 100644
--- a/src/egl/drivers/dri2/egl_dri2.c
+++ b/src/egl/drivers/dri2/egl_dri2.c
@@ -628,7 +628,7 @@ dri2_setup_screen(_EGLDisplay *disp)
       if (dri2_renderer_query_integer(dri2_dpy,
                                       __DRI2_RENDERER_HAS_TEXTURE_3D))
          disp->Extensions.KHR_gl_texture_3D_image = EGL_TRUE;
-#ifdef HAVE_LIBDRM
+#if defined(HAVE_LIBDRM) || defined(HAVE_SURFACELESS_PLATFORM)
       if (dri2_dpy->image->base.version >= 8 &&
           dri2_dpy->image->createImageFromDmaBufs) {
          disp->Extensions.EXT_image_dma_buf_import = EGL_TRUE;
-- 
2.5.1

